Dockerfile

```
@@ -16,6 +17,7 @@ COPY . ./
 RUN go build -ldflags="-s -w \
     -X 'github.com/open-sauced/pizza-cli/pkg/utils.Version=${VERSION}' \
     -X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=${SHA}'" \
```

You need to remove the string termination here for the `ldflags` flag. Otherwise it fails during container build when it runs the `go build` command.

```suggestion
    -X 'github.com/open-sauced/pizza-cli/pkg/utils.Sha=${SHA}' \
```